pkgs/tools/text/cconv/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, autoconf, automake, libtool }:
+let version = "0.6.3"; in
+  stdenv.mkDerivation {
+  name = "cconv-${version}";
+  
+  src = fetchurl {
+    url = "https://github.com/xiaoyjy/cconv/archive/v${version}.tar.gz";
+    sha256 = "82f46a94829f5a8157d6f686e302ff5710108931973e133d6e19593061b81d84";
+  };
+
+  buildInputs = [ autoconf automake libtool ];
```

Comment:
please use `nativeBuildInputs = [ autoreconfHook ];` instead of this and the `autogen.sh` below.